controllers/openstackdataplanerole_controller.go

```diff
@@ -129,6 +129,10 @@ func (r *OpenStackDataPlaneRoleReconciler) Reconcile(ctx context.Context, req ct
 			return ctrl.Result{}, err
 		}
 	}
+	err = instance.Validate(nodes.Items)
```

Comment:
ditto, a validation error for a node, should not be locking out the whole role from futher reconciliation?